prettier.novaextension/extension.json

```diff
@@ -54,204 +58,241 @@
 			"children": [
 				{
 					"key": "prettier.module.path",
-					"title": "Prettier module",
-					"description": "Prettier module folder. If none is specified, it defaults to the Prettier installed in the project or, if there is none, the version embedded in the extension. Can be overridden in project settings.",
+					"title": "Prettier Module",
+					"description": "Set a folder for the Prettier module. If not set, it defaults to the build-in Prettier or the module in the project.",
+					"placeholder": "e.g. /opt/homebrew/lib/node_modules/prettier",
 					"type": "path",
 					"allowFiles": false,
 					"allowFolders": true,
 					"default": true
 				},
 				{
 					"key": "prettier.debug.logging",
-					"title": "Log debug information to Extension Console",
+					"title": "Log Debug Information",
+					"description": "Send debug information to the Extension Console.",
 					"type": "boolean",
-					"default": true
+					"default": false
 				}
 			]
 		},
 		{
 			"key": "prettier.default-config.section",
-			"title": "Default Prettier configuration",
-			"description": "Used for remote files or if no Prettier configuration file exists. Can be overridden in project settings.",
+			"title": "Prettier Configuration",
```

Comment:
I think 'Default' here is pretty useful to have. Since it's ignored in any other case.

Reply:
The reason why I removed the Default was, that as soon as a user changes the settings here it differs from the prettier default as described on the Prettier homepage.It is  the default only for Nova but not the default behavior of Prettier. I hope you can follow my logic here. But I can re-add `Default` if you want.